packages/manager/src/features/CloudPulse/Widget/CloudPulseWidget.tsx

```diff
@@ -332,7 +338,7 @@ export const CloudPulseWidget = (props: CloudPulseWidgetProperties) => {
             dotRadius={1.5}
             height={424}
             legendRows={legendRows}
-            loading={isLoading || metricsApiCallError === jweTokenExpiryError} // keep loading until we fetch the refresh token
+            loading={isLoading || isJweTokenFetching} // keep loading until we are trying to fetch the refresh token
```

Comment:
```suggestion
            loading={isLoading || (metricsApiCallError === jweTokenExpiryError) || isJweTokenFetching} // keep loading until we are trying to fetch the refresh token
```

Lets have all condition, i don't see any harm in that as well, so that everyone knows, we are loading because of Token Expired error